README.md

```diff
@@ -1,61 +1,236 @@
+- The simplest and least expensive option is an [S/MIME email certificate](https://www.digicert.com/tls-ssl/compare-secure-email-smime-certificates).  
+- Other options, such as [document signing certificate](https://www.digicert.com/signing/compare-document-signing-certificates) require more rigor (like proving your identity) and cost more.
+
+<div class="review-comment" >
+TO DO: Add details of getting a cert from Digicert https://www.digicert.com/tls-ssl/compare-certificates using a CSR?
+</div>
+
+For testing and demonstration purposes, you can create a self-signed certificate for use as a root CA. The resulting manifests won't be trusted, but you can use it to run the application to see how it works before purchasing a real certificate from a CA.
+
+Follow these steps:
+
+1. Enter this OpenSSL command:
+    ```
+    openssl req -x509 -sha256 \
+    -days 1825 \
+    -newkey rsa:2048 \
+    -keyout rootCA.key \
+    -out rootCA.crt
+    ```
+    This command creates a "fake" root CA key/certificate.  For a detailed explanation, [see below](#understanding-the-openssl-commands).
+1. You'll be prompted to enter and confirm a PEM passphrase.  Then you'll see a message like this.  Respond to the prompts to provide the required information:
+    ```
+    You are about to be asked to enter information that will be incorporated
+    into your certificate request.
+    What you are about to enter is what is called a Distinguished Name or a DN.
+    There are quite a few fields but you can leave some blank
+    For some fields there will be a default value,
+    If you enter '.', the field will be left blank.
+    -----
+    Country Name (2 letter code) [AU]: ...
+    State or Province Name (full name) [Some-State]: ...
+    Locality Name (eg, city) []: ...
+    Organization Name (eg, company) [Internet Widgits Pty Ltd]: ...
+    Organizational Unit Name (eg, section) []: ...
+    Common Name (e.g. server FQDN or YOUR name) []: ...
+    Email Address []: ...
+    ```
+1. Enter this command to sign the CSR with the "fake" CA key:
+    ```
+    openssl x509 -req \
+    -CA rootCA.crt \
+    -CAkey rootCA.key \
+    -in kms-signing.csr \
+    -out kms-signing.crt \
+    -days 365 \
+    -copy_extensions copyall
+    ```
+    For a detailed explanation of the command, [see below](#understanding-the-openssl-commands).<br/>
+    You'll be prompted for the passphrase you entered in the previous step.
+    You'll see a response like this:
+    ```
+    Certificate request self-signature ok
+    subject=O=C2PA Python Demo, CN=John Smith
+    Enter pass phrase for rootCA.key:
+    ```
+
+### Understanding the OpenSSL commands
+
+The [`openssl req -x509`](https://docs.openssl.org/master/man1/openssl-req/) command
+creates a self-signed certificate for use as a root CA.  The other options in the command are:
+
+<div class="review-comment" >
+OpenSSL docs don't show that `-sha256` is supported for `openssl req -x509` command...?
```

Comment:
It may be a CLI flag that doesn't exist. I copied it from StackOverflow. If everything works without it, removing it makes sense to me.